src/CustomerProfile.js

```diff
@@ -0,0 +1,34 @@
+import { useState, useEffect } from "react";
+
+const CustomerProfile = ({ id }) => {
+  const [allCustomerData, setAllCustomerData] = useState();
+
+  function doFetchForCustomerProf() {
+    fetch(`https://cyf-react.glitch.me/customers/${id}`)
+      .then((response) => response.json())
+      .then((data) => setAllCustomerData(data))
+      .catch((error) => console.error(error));
```

Comment:
nit: You can have isLoading and errorMessage here as well similar to the Booking component.